Running tests with Gradle on Windows fails with 'Invalid character in filename' error

By default, Gradle generates XML reports about all tests run, with the name being equal to test's description. Since `cljunit` creates a description that contains `<`, `:` and `>`, and all those characters are not allowed in filenames on Windows, Gradle fails.

### build.gradle

```groovy
buildscript {
    ext.kotlin_version = '1.0.4'
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

plugins {
    id "com.cursive-ide.clojure" version "1.1.0"
}

apply plugin: 'kotlin'
apply plugin: 'idea'
group 'cljunit-test'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
    maven { url "https://clojars.org/repo" }
}

dependencies {
    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
    testCompile "org.clojure:clojure:1.8.0"
    testCompile "junit:junit:4.11"
    testCompile "net.mikera:cljunit:0.5.0"
    testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
}
```

### src/test/clojure/foo.clj

```clojure
(ns foo
  (:use clojure.test))

(deftest failure
  (println "Hello!")
  (is (= 2 3)))
```

### src/test/kotlin/ClojureTests.kt
```kotlin
import mikera.cljunit.ClojureTest

class ClojureTests : ClojureTest () {
    override fun namespaces(): MutableList<String> {
        return mutableListOf("foo")
    }
}
```

### output:

```
Testing started at 02:56 ...
02:56:42: Executing external task 'test'...
:compileKotlin UP-TO-DATE
:compileJava UP-TO-DATE
:compileClojure UP-TO-DATE
:copyMainKotlinClasses UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestKotlin
:compileTestJava UP-TO-DATE
:compileTestClojure
:copyTestKotlinClasses
:processTestResources UP-TO-DATE
:testClasses
:test
Hello!
:test FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':test'.
> Could not generate test report to 'C:\dokumenty\cljunit-test\build\reports\tests'.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
Total time: 11.212 secs
C:\dokumenty\cljunit-test\build\reports\tests\packages\failure  <foo.html (Nazwa pliku, nazwa katalogu lub składnia etykiety woluminu jest niepoprawna)
```

--- a/src/main/java/mikera/cljunit/VarTester.java
+++ b/src/main/java/mikera/cljunit/VarTester.java
@@ -20,12 +20,8 @@ public class VarTester {
 	@SuppressWarnings("unchecked")
 	public VarTester(String ns,String name) {
 		Clojure.require(ns);
-		testVar=RT.var(ns, name);
-		Map<Keyword,Object> meta = (Map<Keyword, Object>) Clojure.META.invoke(testVar);
-		String file = meta.get(FILE).toString();
-		String line = meta.get(LINE).toString();
-		
-		desc=Description.createSuiteDescription(name + "  <"+file+":"+line+">");		
+		testVar=RT.var(ns, name);		
+		desc=Description.createSuiteDescription(ns + '.' + name);		
 	}
 	
 	public void runTest(RunNotifier n) {

